enumerable.rb

```diff
@@ -0,0 +1,119 @@
+# Enumerable method
+module Enumerable
+  def my_each
+    arr = self
+    if arr.is_a?(Array)
+      (0..(arr.length - 1)).each do |i|
+        yield(arr[i])
+      end
+    end
+    arr
+  end
```

Comment:
- #my_each returns an Enumerator if no block is given
- Should work with hashes and ranges